common/src/main/java/com/wynntils/models/containers/ContainerModel.java

```diff
@@ -60,19 +66,21 @@ public final class ContainerModel extends Model {
     private static final StyledText LOBBY_TITLE = StyledText.fromString("Wynncraft Servers");
 
     @Persisted
-    private final Storage<Integer> finalBankPage = new Storage<>(21);
+    private final Storage<Integer> finalBankPage = new Storage<>(1);
 
     @Persisted
-    private final Storage<Integer> finalBlockBankPage = new Storage<>(12);
+    private final Storage<Integer> finalBlockBankPage = new Storage<>(1);
 
     @Persisted
-    private final Storage<Integer> finalBookshelfPage = new Storage<>(10);
+    private final Storage<Integer> finalBookshelfPage = new Storage<>(1);
 
     @Persisted
-    private final Storage<Integer> finalMiscBucketPage = new Storage<>(10);
```

Comment:
I had to change these or else the quickJumpModel might try to jump somewhere it can't endlessly (only an assumption; haven't experienced it yet)

Reply:
This shouldn't be required. With this, you won't be able to jump anywhere until you visit your final page. With them set to the max, you can jump anywhere that's valid and it only doesn't work if your final page is close to a vanilla destination and you have a custom destination there.

For example, if you own 16 pages and try to jump to 16, the closest will be 17 but since you don't own it nothing will happen. It's better than not being able to jump anywhere at all until you visit the final page